scss/_base_links.scss

```diff
@@ -7,7 +7,9 @@
     text-decoration: none;
 
     &:focus {
-      outline: thin dotted $color-mid-light;
+      @include vf-focus;
```

Comment:
`vf-focus` adds `&:focus` by itself, so it shouldn't be nested in `:focus` selector.